packages/remix/src/index.server.ts

```diff
@@ -119,10 +126,36 @@ export { ErrorBoundary, withErrorBoundary } from '@sentry/react';
 export { withSentry } from './client/performance';
 export { captureRemixErrorBoundaryError } from './client/errors';
 export { browserTracingIntegration } from './client/browserTracingIntegration';
-export { wrapExpressCreateRequestHandler } from './utils/serverAdapters/express';
 
 export type { SentryMetaArgs } from './utils/types';
 
+/**
+ * Returns the default Remix integrations.
+ *
+ * @param options The options for the SDK.
+ */
+export function getDefaultIntegrations(options: RemixOptions): Integration[] {
+  return [
+    ...getDefaultNodeIntegrations(options).filter(integration => integration.name !== 'Http'),
+    httpIntegration(),
```

Comment:
Let's refactor this to something like:

```js
function getRemixDefaultIntegrations() {
  return [...]
}

// further down:

if (options.autoInstrumentRemix) {
  options.defaultIntegrations = getRemixDefaultIntegrations(options);
} else {
  instrumentServer();
}
```

This way users can opt in to the new behavior, but keep the old behavior working if needed?

This also means that `wrapExpressCreateRequestHandler` should probably remain as it was before, users need to opt-in to not use that anymore...?

Reply:
Skipping incoming requests worked well both on legacy and otel implementations. I think I managed to keep the old behaviour, without requiring `wrapExpressCreateRequestHandler`, creating `http.server` span manually.
Dropping a sample event [here](https://sentry-sdks.sentry.io/performance/trace/2d7da0fba10f3745a98c484bafb1014e/?node=txn-da1e1ef946c74d4aba70f113dab3dd4e&statsPeriod=7d&timestamp=1718103519)